CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+  - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
+
+3. Install or update to **Node.js v16**. For more information, see [the development guide](contributing/development.md).
+
+4. Create a working branch and start with your changes!
+
+### Commit your update
+
+Commit the changes once you are happy with them. Don't forget to [self-review](#self-review) to speed up the review process:zap:.
+
+### Self review
+
+You should always review your own PR first.
+
+For content changes, make sure that you:
+
+- [ ] Confirm that the changes meet the user experience and goals outlined in the content design plan (if there is one).
+- [ ] Compare your pull request's source changes to staging to confirm that the output matches the source and that everything is rendering as expected. This helps spot issues like typos, content that doesn't follow the style guide, or content that isn't rendering due to versioning problems. Remember that lists and tables can be tricky.
+- [ ] Review the content for technical accuracy.
+- [ ] Review the entire pull request using the [translations guide for writers](./translations/for-writers.md).
+- [ ] Copy-edit the changes for grammar, spelling, and adherence to the [style guide](#).
+- [ ] Check new or updated Liquid statements to confirm that versioning is correct.
+- [ ] If there are any failing checks in your PR, troubleshoot them until they're all passing.
+
+
+### Pull Request
+
+When you're finished with the changes, create a pull request, also known as a PR.
+- Fill the "Ready for review" template so that we can review your PR. This template helps reviewers understand your changes as well as the purpose of your pull request. 
+- Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
+- Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
+Once you submit your PR, a Docs team member will review your proposal. We may ask questions or request additional information.
+- We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes in your fork, then commit them to your branch.
+- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
```

Comment:
This is asking too much I guess, most probably no one will do it

Reply:
I think even if only a few do it, it would still be a bit of a time saver. If they still decide not to then it also doesn't matter. 
@srcnalt what you think?

Reply:
Yes since this is a guideline it can be a bit educational too, I don't see any issues with that. However we can compress it a little more, certain things felt similar/repetitive to me.

Reply:
I think I'll just leave for now. We can do a 2nd pass on these later (maybe our docs guy can take a look over it)